lib/meadow/data.ex

```diff
@@ -0,0 +1,86 @@
+defmodule Meadow.Data do
```

Comment:
I think if we want to follow our recent contexts ADR, most of these would belong in /lib/meadow/data/works.ex